selfdrive/car/nissan/carstate.py

```diff
@@ -49,13 +49,16 @@ def update(self, cp, cp_adas, cp_cam):
     if self.CP.carFingerprint in [CAR.ROGUE, CAR.XTRAIL]:
       ret.seatbeltUnlatched = cp.vl["HUD"]["SEATBELT_DRIVER_LATCHED"] == 0
       ret.cruiseState.available = bool(cp_cam.vl["PRO_PILOT"]["CRUISE_ON"])
-    elif self.CP.carFingerprint == CAR.LEAF:
-      ret.seatbeltUnlatched = cp.vl["SEATBELT"]["SEATBELT_DRIVER_LATCHED"] == 0
+    elif self.CP.carFingerprint in [CAR.LEAF, CAR.LEAF_2_ZERO]:
+      if self.CP.carFingerprint == CAR.LEAF:
+        ret.seatbeltUnlatched = cp.vl["SEATBELT"]["SEATBELT_DRIVER_LATCHED"] == 0
+      # TODO: fix this for LEAF_2_ZERO
```

Comment:
Did you find the seatbelt signal for this car?

Reply:
Yes, just yesterday. We checked every bit for 0/1 correlation and only found 0:640 and 130:640 to be good. Everything else was far off. Just pushed it.

Reply:
Also if this helps, the last bit of the route contains a clip where the car is in the garage. Everytime I cover the camera with my hand, the seatbelt is latched. I repeat this 5-8 times. This is how we found this signal to be good.